modules/ROOT/pages/troubleshoot-guide.adoc

```diff
@@ -45,39 +45,29 @@ tcp://dias-ingestor-nginx.prod.cloudhub.io:5044 ✔
  
 If you have outbound connectivity issues that prevent Runtime Fabric from reaching any of the required Anypoint control plane services, work with your network team to verify that you have completed the required whitelisting as described in xref:install-port-reqs#port-ips-and-hostnames-to-whitelist[Port IPs and Hostnames to Whitelist].
 
-== Resuming a Failed Installation
+== Troubleshoot Cluster Issues
 
-You can resume an installation at the point where it failed by running the init script:
+When filing a support case, the support team might ask you to run one or both of the following commands to generate debugging information:
 
-* AWS and manual installations:
+* The `rtfctl report` command to generate an archive containing only Kubernetes objects and logs.
+* The `rtfctl appliance report` command to collect diagnostics from all cluster nodes.
 
-+
-----
-/opt/anypoint/runtimefabric/init.sh
-----
+The support team might also ask you to download information through Ops Center as described in xref:using-opscenter#download-debug-info[Download Debug Info].
 
-* Azure installations:
-+
-----
-/opt/anypoint/runtimefabric/script.sh
-----
-
-== Troubleshooting Application Deployment Issues
+== Troubleshoot Application Deployment Issues
 
 In rare situations, the Anypoint Monitoring agent might cause an application to not deploy. In
```

Comment:
might cause an application to not deploy > might prevent an application from deploying

Reply:
Good catch - thank you!